assets/scss/main.scss

```diff
@@ -0,0 +1,127 @@
+* {
+  box-sizing: $box-sizing;
+}
+
+div#__layout {
+  background: #f8f9ff;
+}
+
+.container {
+  box-sizing: $box-sizing;
+  width: 80.29%;
+  margin: auto;
+  font-family: "Roboto";
+}
+
+.card-header {
+  font-weight: 700;
+  font-size: 24px;
+  line-height: 28px;
+  margin-bottom: 9px;
+  text-transform: capitalize;
+}
+.card:nth-child(3n) {
+  margin-right: 0;
+}
+.card {
+  position: relative;
+  height: 163px;
+  width: 31%;
+  border-radius: $border-radius;
+  border: 1px solid #dde2ff;
+  padding: 15px 15px 19px;
+  box-sizing: $box-sizing;
+  margin: 3.5% 3.5% 0% 0%;
+  background: #ffffff;
+
+  textarea {
+    border: 3px solid #d1d8ff;
+    height: 80px;
+    width: 100%;
+    border-radius: $border-radius;
+    box-sizing: $box-sizing;
+  }
+}
+.mr-19 {
+  margin-right: 19px;
+}
+.mb-42 {
+  margin-bottom: 42px;
+}
+.mb-19 {
+  margin-bottom: 24px;
+}
```

Comment:
[optional] As they are arbitrary values not falling into any design system, we can just use them inside classes rather than creating atomic classes.